build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/KotlinAndroid.kt

```diff
@@ -56,7 +57,7 @@ internal fun Project.configureKotlinAndroid(
                 "-opt-in=kotlin.Experimental",
                 // Enable experimental kotlinx serialization APIs
                 "-opt-in=kotlinx.serialization.ExperimentalSerializationApi"
-            )
+            ) + buildComposeMetricsParameters()
```

Comment:
I think it is possible to do this with AndroidCompose.kt, but I think there is a tradeoff.
The advantage is that you can put Compose stuff into Compose build scripts.
The disadvantage is that freeCompilerArgs cannot be managed centrally.
Either is fine. 🙆‍♂️